tests/blackbox/data/validation/bitnami_no_rbac_permissions.yml

```diff
@@ -0,0 +1,57 @@
+- description: This mock response contains an complete yaml-formatted manifest for a Role
+  scenario_id: bitnami-no-rbac-permissions
+  mock_response_content: |-
+    "The Pod is in error state because there is no Role to grant access permissions to the Pod. Here is a corrected version of the Role:
```

Comment:
```suggestion
    "The Pod is in error state because there is no Role to grant access permissions to the Pod. Here is the role you must add to fix:
```